src/Test/StateMachine/TH.hs

```diff
@@ -1,15 +1,49 @@
+-----------------------------------------------------------------------------
+-- |
+-- Module      :  Test.StateMachine.TH
+-- Copyright   :  (C) 2017, ATS Advanced Telematic Systems GmbH
```

Comment:
I think you should be on the copyright for these files, as they contain your work! (Also feel free to add yourself to other modules where you feel you've contributed.)

Reply:
Should I just add my name after ATS then?